Transfer/Sources/ScheduleTransferController.swift

```diff
@@ -130,6 +174,14 @@ extension ScheduleTransferController {
     private func tapScheduleTransfer(sender: UITapGestureRecognizer) {
         presenter.scheduleTransfer()
     }
+
+    func addFooter() {
```

Comment:
No need of this

Reply:
You mean put this code in the switch statement itself - inside `getCellConfiguration`

Reply:
No we do not need it at all. ``` override public func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView?  ``` will add the footer. Also fix all the swiftlint warning too